lib/ansible/plugins/inventory/vmware_vm_inventory.py

```diff
@@ -31,12 +31,12 @@
             description: Name of vCenter or ESXi server.
             required: True
             env:
-              - name: VMWARE_SERVER
+              - name: VMWARE_HOST
         username:
             description: Name of vSphere admin user.
             required: True
             env:
-              - name: VMWARE_USERNAME
+              - name: VMWARE_USER
```

Comment:
These env vars cannot be simply changed, both would need supported, potentially deprecating another.

Reply:
Added VMWARE_USER as environment variable to match value used by tower and vmware_* modules